ui/src/app/pages/sinks/add/sink-add.component.ts

```diff
@@ -79,14 +79,7 @@ export class SinkAddComponent {
 
         return !this.editor.checkEmpty(config.authentication)
         && !this.editor.checkEmpty(config.exporter)
-        && detailsValid
-        && !this.checkString(config);
-    }
-    checkString(config: any): boolean {
-        if (typeof config.authentication.password !== 'string' || typeof config.authentication.username !== 'string') {
-            return true;
-        }
-        return false;
+        && detailsValid;
```

Comment:
It no longer makes sense to disable the `create` button when we don't have a username and password.
If token or password is missing we will get 400 response.